.circleci/config.yml

```diff
@@ -165,6 +165,7 @@ jobs:
           name: filter out DONTCOVER
           command: |
             excludelist="$(find ./ -type f -name '*.go' | xargs grep -l 'DONTCOVER')"
+            excludelist+="$(find ./ -name '*.pb.go')"
```

Comment:
this should be fixed in the codecov.yml file, this is a hack more or less

Reply:
https://github.com/cosmos/cosmos-sdk/blob/master/.codecov.yml#L49

For some reason these files are still reported as untested, e.g.:

https://codecov.io/gh/cosmos/cosmos-sdk/src/master/types/types.pb.go

Reply:
yea there is a weird bug with that. In this PR https://github.com/cosmos/cosmos-sdk/pull/5493 it seems to have done it correctly. maybe we wait to merge that to see what happens. 

also in the next release of protogen-go-gocosmos it will default to create the test files for the autogenerated files

Reply:
Ok that makes total sense. Hence closing. Thanks!